### DISCUSSION TITLE
How to find link to rices in `Hall of fame`?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [hyprwm/Hyprland#3857] [Q&A]
https://hyprland.org/rices

There are only authors name and one screenshot per each.

**Comment by vaxerski** [comment]
the website is being rewritten (ref https://github.com/hyprwm/hyprland-website/pull/28 )

in the rewrite there will be links.

If you wanna see a live version of it, https://hyprtesty.surge.sh/

Go to WoF and you can click the names to see the dots.

**Comment by KAGEYAM4** [comment]
the new website looks awesome. Thanks
